kirby.php

```diff
@@ -491,20 +509,23 @@ public function extensions() {
    */
   public function models() {
 
-    if(!is_dir($this->roots()->models())) return false;
+    $roots   = $this->modules()->allRoots('models', $this->roots()->models());
+    $load    = array();
+
+    foreach($roots as $root) {
+      if(!is_dir($root)) continue;
 
-    $root  = $this->roots()->models();
-    $files = dir::read($root);
-    $load  = array();
+      $files = dir::read($root);
 
-    foreach($files as $file) {
-      if(f::extension($file) != 'php') continue;
-      $name      = f::name($file);
-      $classname = str_replace(array('.', '-', '_'), '', $name . 'page');
-      $load[$classname] = $root . DS . $file;
+      foreach($files as $file) {
+        if(f::extension($file) != 'php') continue;
+        $name      = f::name($file);
+        $classname = str_replace(array('.', '-', '_'), '', $name . 'page');
+        $load[$classname] = $root . DS . $file;
```

Comment:
The last registered directory will win here and will override models defined elsewhere. Maybe the directory in `site/models` should be preferred so that users can override the models. I honestly don't know.